### PR TITLE
Move log message out of for loop

### PR DESCRIPTION
**Pull request by gigawhitlocks**
This informational log message should only get printed once, and I erroneously put it inside of the for loop below where it should be.

This PR fixes that. Also adds some traces for debug.

### CODE REVIEW

**Comment by emlin** [comment]
lgtm

File: lib/portlayer/storage/vsphere/datastore.go [hunk 1]
```diff
@@ -311,6 +312,7 @@ var pathFormat = regexp.MustCompile(`\s([\/\w]+$)`)
 
 // Converts `[datastore] /path` to URL
 func DatastoreToURL(ds string) (*url.URL, error) {
+	defer trace.End(trace.Begin(""))
```
Comment (caglar10ur) [inline lib/portlayer/storage/vsphere/datastore.go]: use ds instead of ""

File: lib/portlayer/storage/vsphere/datastore.go [hunk 2]
```diff
@@ -329,6 +331,7 @@ func DatastoreToURL(ds string) (*url.URL, error) {
 
 // Converts URL for datastores to datastore format ([datastore1] /path/to/thing)
 func URLtoDatastore(u *url.URL) (string, error) {
+	defer trace.End(trace.Begin(""))
```
Comment (caglar10ur) [inline lib/portlayer/storage/vsphere/datastore.go]: use u.String() instead of ""

**Comment by caglar10ur** [comment]
just cosmetic nitpicks LGTM otherwise

**Comment by gigawhitlocks** [comment]
Thanks @caglar10ur

**Comment by gigawhitlocks** [comment]
Poke

**Pull request merged**